lib/template-parts.php

```diff
@@ -44,12 +44,14 @@ function gutenberg_register_template_part_post_type() {
 		'show_in_menu'      => 'themes.php',
 		'show_in_admin_bar' => false,
 		'show_in_rest'      => true,
+		'rest_base'         => 'template-parts',
 		'map_meta_cap'      => true,
 		'supports'          => array(
 			'title',
 			'slug',
 			'editor',
 			'revisions',
+			'custom-fields',
```

Comment:
Not directly related to this PR, but I'm curious about why a custom field was used, would it make sense to use a custom taxonomy? For example, the navigation menu CPT structure uses custom taxonomies.

Reply:
I guess in this case it's a tradeoff between being able to do sorted queries by theme value (custom fields) vs. the better performance of queries for a specific theme value (taxonomy).

I think that it doesn't make a difference for the number of posts we are dealing with for this post type.